config/groovy/util.groovy

```diff
@@ -220,6 +220,36 @@ switch (cleanerArgs[0]) {
 
         break
 
+        // Makes sure a workspace has one or more modules matching user input, most useful for grabbing whole "module distros"
+        case "init":
+            // TODO: Move most this code into module.groovy, leave an error here if you use it for something else
+            if (cleanerArgs.length == 1) {
+                println "Doing a default module init - simply fetching CoreSampleGameplay without dependencies in source (same as `groovyw module get CoreSampleGameplay`)"
+                String[] targetMods = ["CoreSampleGameplay"]
+                common.retrieve(targetMods, false)
+            } else if (cleanerArgs.length == 2) {
+                def targetModuleDistro = cleanerArgs[1]
+                println "Doing an init of a given module distro: " + targetModuleDistro
+                def targetDistroURL = "https://raw.githubusercontent.com/Terasology/Index/master/distros/" + targetModuleDistro + "/gradle.properties"
+                if (!common.isUrlValid(targetDistroURL)) {
+                    println "Distro URL $targetDistroURL does not seem to be valid, typo? See https://github.com/Terasology/Index/tree/master/distros for available distros"
```

Comment:
If something goes wrong, this prints a very verobse message (imho)

> Distro URL https://raw.githubusercontent.com/Terasology/Index/master/distros/jota/gradle.properties does not seem to be valid, typo? See https://github.com/Terasology/Index/tree/master/distros for available distros


I would slim this down and split it up a bit:

```suggestion
                    println "[init] Invalid distribution name: '$targetModuleDistro'"
                    println "[init]     See https://github.com/Terasology/Index/tree/master/distros for available distributions
```

As follow-up improvement, we could just fetch the list of supported distros dynamically:

> ```
> [init] Invalid distribution name: "jota"
> [init]     Available distributions: [awt, iota, omega]
> ```